dev/integration_tests/android_embedding/README.md

```diff
@@ -0,0 +1,16 @@
+# android_embedding
```

Comment:
Can we update this with desriptive narrative of what this folder is for?  Perhaps just your message in the PR comment?

Reply:
Done